src/lib/features/frontend-api/frontend-api-controller.ts

```diff
@@ -186,48 +183,11 @@ export default class FrontendAPIController extends Controller {
         if (!this.config.flagResolver.isEnabled('embedProxy')) {
             throw new NotFoundError();
         }
-        let toggles: FrontendApiFeatureSchema[];
-        let newToggles: FrontendApiFeatureSchema[] = [];
-        if (this.config.flagResolver.isEnabled('globalFrontendApiCache')) {
-            const context = FrontendAPIController.createContext(req);
-            [toggles, newToggles] = await Promise.all([
-                this.getTimedFrontendApiFeatures(req, context),
-                this.getTimedNewFrontendApiFeatures(req, context),
-            ]);
-            const sortedToggles = toggles.sort((a, b) =>
-                a.name.localeCompare(b.name),
-            );
-            const sortedNewToggles = newToggles.sort((a, b) =>
-                a.name.localeCompare(b.name),
+        const toggles =
+            await this.services.frontendApiService.getFrontendApiFeatures(
+                req.user,
+                FrontendAPIController.createContext(req),
             );
-            if (!isEqual(sortedToggles, sortedNewToggles)) {
-                this.logger.warn(
-                    `old features and new features are different. Old count ${
-                        toggles.length
-                    }, new count ${newToggles.length}, projects ${
-                        req.user.projects
-                    }, environment ${
-                        req.user.environment
-                    }, diff ${JSON.stringify(
-                        diff(sortedToggles, sortedNewToggles),
-                    )}`,
-                );
-            }
-        } else if (
-            this.config.flagResolver.isEnabled('returnGlobalFrontendApiCache')
-        ) {
-            toggles =
```

Comment:
just to make sure, is this the path that we return after the flag removal?

Reply:
yes